roles/wildfly_systemd/tasks/main.yml

```diff
@@ -17,6 +17,14 @@
     name: wildfly_install
     tasks_from: user.yml
 
+- name: Ensure PID directory exists
```

Comment:
This should only execute if the directory does not exist, or otherwise it will set the wildfly user to own it (ie. in the default case, it make /var/run owned by wildfly_user)

Reply:
Thanks! changed it accordingly.